Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -163,6 +163,19 @@ initData = {
     , isMockLocation: false
     , isSpecialZone : false
     , defaultPickUpPoint : ""
+    , tipViewProps : {
+        isVisible : false
+      , onlyPrimaryText : false
+      , isprimaryButtonVisible : false
+      , primaryText : "Add a tip to find a ride quicker!"
+      , secondaryText : "It seems to be taking longer than usual."
```

Comment:
don't add any string in initData keep it empty

Reply:
done